OpenRA.Mods.Common/UtilityCommands/ExtractSequenceDocsCommand.cs

```diff
@@ -48,7 +48,10 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 
 		static string GenerateJson(string version, IEnumerable<Type> sequenceTypes)
 		{
-			var sequenceTypesInfo = sequenceTypes.Where(x => !x.ContainsGenericParameters && !x.IsAbstract)
+			var relatedEnumTypes = new HashSet<Type>();
+
+			var sequenceTypesInfo = sequenceTypes
+				.Where(x => !x.ContainsGenericParameters && !x.IsAbstract)
 				.Where(x => x.Name != nameof(FileNotFoundSequence)) // NOTE: This is the simplest way to exclude FileNotFoundSequence, which shouldn't be added.
```

Comment:
this should really be reusing the .Where

Reply:
I think this is much better for readability and for the comment that follows though.

Reply:
it can stay in new line, won't make it any less readable